docs/docs/external-plugins.md

```diff
@@ -3,6 +3,8 @@ title: External Plugins
 ---
 
 - [Adding an external plugin](#adding-an-external-plugin)
+- [Example Plugins](#example-plugins)
```

Comment:
Reason for "move-up": makes it easier to complement user's thinking of "I want to use Wash for its external plugins => Ok, here's how I can add one => Oh, these are some plugins people have built, can I re-use any? OR Oh, there are some libraries to make this easy. Sweet"